draft-ietf-quic-multipath.md

```diff
@@ -558,25 +568,31 @@ can send ack-eliciting packets such as packets containing PING frames
 periodic PING frames also helps prevent middlebox timeout, as discussed in
 {{Section 10.1.2 of QUIC-TRANSPORT}}.
 
-Server MAY release the resources associated with a path for
-which no non-probing packet was received for a sufficiently long
-path-idle delay, but SHOULD only release resources for the last
-available path if no traffic is received for the duration of the idle
-timeout, as specified in {{Section 10.1 of QUIC-TRANSPORT}}.
-This means if all paths remain idle for the idle timeout, the connection
-is implicitly closed.
-
 Server implementations need to select the sub-path idle timeout as a trade-
 off between keeping resources, such as connection IDs, in use
 for an excessive time or having to promptly re-establish a path
 after a spurious estimate of path abandonment by the client.
 
+Endpoints that desire to close a path because of the idle timer rule
+MUST do so explicitly by sending a PATH_ABANDON frame, as defined in
```

Comment:
```suggestion
MUST do so explicitly by sending a PATH_ABANDON frame on another active path, as defined in
```

Reply:
I think it's better to allow endpoints to send PATH_ABANDON frame either on the same path or another active path. This depends on whether the same path is a blackhole. So we don't need to specify here.